stock_picking_package_info/models/product_product.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    weight_net = fields.Float(
+        string='Net Weight',
+        related='product_tmpl_id.weight_net',
```

Comment:
We should have this weight at product variant level, not at template level, but when setting at template level, it should be propagated to the variants.